openshift/catalogd/kustomize/overlays/openshift/olmv1-ns/metrics/catalogd_metrics_monitor_servicemonitor.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: metrics-monitor
+  labels:
+    app.kubernetes.io/name: catalogd
+    app.kubernetes.io/part-of: olm
+    openshift.io/cluster-monitoring: 'true'
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      path: /metrics
+      port: metrics
```

Comment:
What is this value (numerically)? Does it line up with the catalogd service (`7443`)?

Reply:
Good catcher. That cannot work without we set the port.